src/Eloquent/Builder.php

```diff
@@ -153,14 +156,6 @@ public function decrement($column, $amount = 1, array $extra = [])
         return parent::decrement($column, $amount, $extra);
     }
 
-    /**
-     * @inheritdoc
-     */
-    public function chunkById($count, callable $callback, $column = '_id', $alias = null)
```

Comment:
Laravel 10 uses [`defaultKeyName`](https://github.com/laravel/framework/blob/85ad68cb0bcb5143e0e915098097c6d23ea22e7e/src/Illuminate/Database/Eloquent/Builder.php#L1796) to get de model key name.

Reply:
https://github.com/GromNaN/laravel-mongodb/pull/25